skyrim-platform/src/platform_se/codegen/TSConverter.cpp

```diff
@@ -86,6 +86,7 @@ int main(int argc, char* argv[])
 
   auto parseReturnValue = [&](std::string rawType,
                               std::string objectTypeName) -> std::string {
+
```

Comment:
Could you please remove this whitespace?

Reply:
I think it's going to be merged after that. LGTM.

Reply:
Ready